<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMessage</class>
 <widget class="QDialog" name="UserMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#UserMessage {
	background-color: rgb(255, 255, 255);
	border: 1px solid #CCCCCC;
	margin: 0 0 0 0;
}</string>
  </property>
  <widget class="QTextBrowser" name="userMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>360</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>360</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#messageField{
	color: #333333;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConfirm">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pbConfirm {
    background-color: #00c38f;
    border-color: #00c38f;
    border-style: solid;
    color: #ffffff;
    border-width: 1px;
}

#pbConfirm:hover {
    background-color: #00aa7d;
}

#pbConfirm:pressed {
    background-color: #008c67;
}</string>
   </property>
   <property name="text">
    <string>Да</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pbCancel {
    background-color: #00c38f;
    border-color: #00c38f;
    border-style: solid;
    color: #ffffff;
    border-width: 1px;
}

#pbCancel:hover {
    background-color: #00aa7d;
}

#pbCancel:pressed {
    background-color: #008c67;
}</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
